desktop/libs/aws/src/aws/s3/s3connection.py

```diff
@@ -160,6 +160,12 @@ def get_signed_url(self, action='GET', url=None, headers=None, data=None):
 
     return raz_client.get_url(action, url, headers, data)
 
+  def _required_auth_capability(self):
```

Comment:
Question: Why not simply return `return ['anon']` because this RAZS3Connection will only be used by the S3Filesystem client when RAZ is enabled.
cf: https://github.com/cloudera/hue/blob/84e8e4231822c79816c458585e9fae7ddb392b53/desktop/libs/aws/src/aws/client.py#L47-L54

Reply:
Makes sense. I updated the PR.

Reply:
Suggestion: Maybe also add a docstring or comment to explain why we are overriding this method?

Reply:
done